assets/xml/objects/object_dnq.xml

```diff
@@ -1,88 +1,99 @@
-        <Limb Name="object_dnq_Standardlimb_010C24" Type="Standard" EnumName="OBJECT_DNQ_LIMB_08" Offset="0x10C24" />
-        <Limb Name="object_dnq_Standardlimb_010C30" Type="Standard" EnumName="OBJECT_DNQ_LIMB_09" Offset="0x10C30" />
-        <Limb Name="object_dnq_Standardlimb_010C3C" Type="Standard" EnumName="OBJECT_DNQ_LIMB_0A" Offset="0x10C3C" />
-        <Limb Name="object_dnq_Standardlimb_010C48" Type="Standard" EnumName="OBJECT_DNQ_LIMB_0B" Offset="0x10C48" />
-        <Limb Name="object_dnq_Standardlimb_010C54" Type="Standard" EnumName="OBJECT_DNQ_LIMB_0C" Offset="0x10C54" />
-        <Limb Name="object_dnq_Standardlimb_010C60" Type="Standard" EnumName="OBJECT_DNQ_LIMB_0D" Offset="0x10C60" />
-        <Limb Name="object_dnq_Standardlimb_010C6C" Type="Standard" EnumName="OBJECT_DNQ_LIMB_0E" Offset="0x10C6C" />
-        <Limb Name="object_dnq_Standardlimb_010C78" Type="Standard" EnumName="OBJECT_DNQ_LIMB_0F" Offset="0x10C78" />
-        <Limb Name="object_dnq_Standardlimb_010C84" Type="Standard" EnumName="OBJECT_DNQ_LIMB_10" Offset="0x10C84" />
-        <Limb Name="object_dnq_Standardlimb_010C90" Type="Standard" EnumName="OBJECT_DNQ_LIMB_11" Offset="0x10C90" />
-        <Limb Name="object_dnq_Standardlimb_010C9C" Type="Standard" EnumName="OBJECT_DNQ_LIMB_12" Offset="0x10C9C" />
-        <Limb Name="object_dnq_Standardlimb_010CA8" Type="Standard" EnumName="OBJECT_DNQ_LIMB_13" Offset="0x10CA8" />
-        <Limb Name="object_dnq_Standardlimb_010CB4" Type="Standard" EnumName="OBJECT_DNQ_LIMB_14" Offset="0x10CB4" />
-        <Limb Name="object_dnq_Standardlimb_010CC0" Type="Standard" EnumName="OBJECT_DNQ_LIMB_15" Offset="0x10CC0" />
-        <Limb Name="object_dnq_Standardlimb_010CCC" Type="Standard" EnumName="OBJECT_DNQ_LIMB_16" Offset="0x10CCC" />
-        <Limb Name="object_dnq_Standardlimb_010CD8" Type="Standard" EnumName="OBJECT_DNQ_LIMB_17" Offset="0x10CD8" />
-        <Limb Name="object_dnq_Standardlimb_010CE4" Type="Standard" EnumName="OBJECT_DNQ_LIMB_18" Offset="0x10CE4" />
-        <Limb Name="object_dnq_Standardlimb_010CF0" Type="Standard" EnumName="OBJECT_DNQ_LIMB_19" Offset="0x10CF0" />
-        <Skeleton Name="object_dnq_Skel_010D60" Type="Flex" LimbType="Standard" LimbNone="OBJECT_DNQ_LIMB_NONE" LimbMax="OBJECT_DNQ_LIMB_MAX" EnumName="ObjectDnpLimb" Offset="0x10D60" />
-        <Animation Name="object_dnq_Anim_0115B8" Offset="0x115B8" />
-        <Animation Name="object_dnq_Anim_012428" Offset="0x12428" />
+        <!-- Deku Princess Animations -->
+        <Animation Name="gDekuPrincessJumpKickAnim" Offset="0x7D8" /> <!-- Original name is "dnq_atack" -->
+        <Animation Name="gDekuPrincessJumpToKingAnim" Offset="0x125C" /> <!-- Original name is "dnq_attack1" -->
+        <Animation Name="gDekuPrincessBounceOnKingStartAnim" Offset="0x17F8" /> <!-- Original name is "dnq_attack2" -->
+        <Animation Name="gDekuPrincessBounceOnKingLoopAnim" Offset="0x1C1C" /> <!-- Original name is "dnq_attack2_loop" -->
```

Comment:
Hmm, not 100% sure but I feel like we could possibly drop the `ToKing` and `OnKing` in these animations.